test/e2e/utils.go

```diff
@@ -548,3 +549,53 @@ func deletePersistentVolumeClaims(namespace string) {
 		fw.KubeClient.CoreV1().PersistentVolumeClaims(kafkaNamespace).Delete(pvc.Name, &emptyDeleteOptions)
 	}
 }
+
+func verifyCollectorImage(jaegerInstanceName, namespace string, expected bool) {
+	if expected {
+		require.True(t, wasUsingOtelCollector(jaegerInstanceName, namespace))
+	} else {
+		require.False(t, wasUsingOtelCollector(jaegerInstanceName, namespace))
+	}
+}
+
+// Was this Jaeger Instance using the OTEL collector?
+func wasUsingOtelCollector(jaegerInstanceName, namespace string) bool {
+	deployment, err := fw.KubeClient.AppsV1().Deployments(namespace).Get(jaegerInstanceName+"-collector", metav1.GetOptions{})
+	require.NoError(t, err)
+	containers := deployment.Spec.Template.Spec.Containers
+	for _, container := range containers {
+		if container.Name == "jaeger-collector" {
+			logrus.Infof("Test %s is using image %s", t.Name(), container.Image)
+			return strings.Contains(container.Image, "jaeger-opentelemetry-collector")
+		}
+	}
+
+	require.Failf(t, "Did not find a collector image for %s in namespace %s", jaegerInstanceName, namespace)
+	return false
+}
+
+// Returns a collector spec which uses the Open Telementry Collector
+func getOtelCollectorSpec(collectorSpec v1.JaegerCollectorSpec) v1.JaegerCollectorSpec {
```

Comment:
This is not ideal either change the signature to accept a pointer of the collector spec or return just otel config,  image and options as 3 return values.